.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,25 @@
+---
+name: 🐛 Reporting a Bug
+about: Open a new issue here if something isn't working as expected.
+---
+
+<!--
+  Thanks for filing an issue!
+
+  Please make sure that you include the following information to ensure that your issue is actionable.
+-->
+
```

Comment:
I think we should probably include the following fields from the old issue template here:

```markdown
**Browser name:** 	

**Browser version:** 	

**OctoLinker version:** 	

**URL and line number where issue occurs:**
```